### ISSUE TITLE
Accessibility - Alt Text Missing

### ISSUE THREAD

**Issue by SteveLindsay461** [MHRA/products#194]
Raised by Kyle
**Describe the bug**
There is no alt text showing on the ‘Search’, ‘Next’ or ‘Previous’ buttons

**To Reproduce**

To Replicate: Hove over the Search, Previous and Next buttons and no alternative text is displayed

**Expected behavior**
Appropriate alternative text is displayed when the user hovers over a button

**Screenshots**
![image](https://user-images.githubusercontent.com/58432596/70812976-980aea80-1dc0-11ea-9212-eaddbb9608bd.png)
![image](https://user-images.githubusercontent.com/58432596/70812987-9e00cb80-1dc0-11ea-8afc-372e618ec043.png)

**Additional context**

**Comment by craiga** [comment]
@SteveLindsay461 I'm unsure what the alternative text here should say. Do you have any suggestions?

**Comment by SteveLindsay461** [comment]
Hi Craig

Kyle is the person you need to confirm with on this.


Steve Lindsay
Service Transition Officer
Transformation Division

Medicines and Healthcare products Regulatory Agency
10 South Colonnade, Canary Wharf, London E14 4PU

Email: steve.lindsay@mhra.gov.uk<mailto:steve.lindsay@mhra.gov.uk>
gov.uk/mhra<https://www.gov.uk/government/organisations/medicines-and-healthcare-products-regulatory-agency>

Stay connected<https://www.gov.uk/government/organisations/medicines-and-healthcare-products-regulatory-agency#what-we-do>
